packages/appstore/src/components/onboarding/static-applauncher.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Icon, Button, Text } from '@deriv/components';
+import { formatMoney } from '@deriv/shared';
+import { Localize, localize } from '@deriv/translations';
+
+import './static-applauncher.scss';
+
+type TStaticAppLauncher = {
+    icon_type: 'USD' | 'Bitcoin' | 'Ethereum' | 'Litecoin';
+    is_item_blurry?: boolean;
+    is_grey?: boolean;
+};
+
+const StaticAppLauncher = ({ icon_type, is_item_blurry, is_grey }: TStaticAppLauncher) => {
+    return (
+        <div
+            className={classNames('static-applauncher', {
+                'static-applauncher--grey': is_grey,
+            })}
+        >
+            <div className='static-applauncher__icon'>
+                {icon_type === 'USD' && <Icon icon='IcCurrencyUsd' is_item_blurry size={38} />}
+                {icon_type === 'Bitcoin' && (
+                    <Icon
+                        icon='IcCurrencyBtc'
+                        is_item_blurry
+                        size={38}
+                        className={'static-applauncher__icon--blurry'}
+                    />
+                )}
+                {icon_type === 'Ethereum' && (
+                    <Icon
+                        icon='IcCurrencyEth'
+                        is_item_blurry
+                        size={38}
+                        className={'static-applauncher__icon--blurry'}
+                    />
+                )}
+                {icon_type === 'Litecoin' && (
+                    <Icon
+                        icon='IcCurrencyLtc'
+                        is_item_blurry
+                        size={38}
+                        className={'static-applauncher__icon--blurry'}
+                    />
+                )}
```

Comment:
You can actually extract these Currency Icons to its own component instead of repeating it this way